.gitignore

```diff
@@ -9,3 +9,6 @@
 *.npy
 *.npz
 .vscode/settings.json
+*.swp
```

Comment:
Is .swp from Docker?

Reply:
Nope, *.swo and *.swp are vim backup files.